chrome/content/coniks/coniks.js

```diff
@@ -0,0 +1,82 @@
+this.EXPORTED_SYMBOLS = ["coniks"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("resource:///modules/imServices.jsm");
+Cu.import("resource:///modules/imXPCOMUtils.jsm");
+Cu.import("resource:///modules/jsProtoHelper.jsm");
+Cu.import("chrome://otr/content/coniks/context.js");
+Cu.import("chrome://otr/content/coniks/policies.js");
+Cu.import("chrome://otr/content/helpers.js");
+
+function accountname(acc) {
+  let account = acc.normalizedName;
+  let protocol = acc.protocol.normalizedName;
+  return account+"@"+protocol;
+}
+
+var coniks = {
+  _isEnable: true,
+  get Enabled() { return coniks._isEnable; },
+
+  init: function() {
+    coniks.setConfig();
+    this._contexts = new Map();
```

Comment:
I think `_contexts` also stores the policies of account's contacts. In the `coniksContext` of each account, there would have an array of his/her contacts' context. The file would be something like:
```
[
  {
    "account": "alice",
    "allowUnsignedKeyChange": false,
    "allowPublicLookup": true,
    "contacts": [
      {
        "account": "bob",
        "allowUnsignedKeyChange": true,
        "allowPublicLookup": false
      },
      {
        "account": "eve",
        "allowUnsignedKeyChange": true,
        "allowPublicLookup": false
      }
    ]
  }
]
```

I didn't think about it thoroughly, though.